tests/test_config.py

```diff
@@ -0,0 +1,32 @@
+import subprocess
+import sys
+import unittest
+from pathlib import Path
+
+import mopidy
+
+
+class SubcommandConfigTest(unittest.TestCase):
```

Comment:
I'd drop the unittest test case for all new tests and just go with plain pytest!